src/Controls/src/Core/Accelerator.cs

```diff
@@ -22,13 +21,14 @@ internal Accelerator(string text)
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/Accelerator.xml" path="//Member[@MemberName='Modifiers']/Docs/*" />
 		[EditorBrowsable(EditorBrowsableState.Never)]
-		public IEnumerable<string> Modifiers { get; set; }
+		public IReadOnlyList<string> Modifiers { get; set; }
 
 		/// <include file="../../docs/Microsoft.Maui.Controls/Accelerator.xml" path="//Member[@MemberName='Keys']/Docs/*" />
 		[EditorBrowsable(EditorBrowsableState.Never)]
+		[Obsolete("Use Key instead.")]
 		public IEnumerable<string> Keys { get; set; }
 
-		string IAccelerator.Key => (Keys is not null) ? Keys.First() : string.Empty;
+		public string Key { get; set; }
```

Comment:
This is interesting because all code that was looking at `Keys` is now no longer going to work. Either it will set by the user and not be used, or a creation happens and this property is blank.

We could make `Keys` update `Key`:

```cs
public IEnumerable<string> Keys
{
    get => Key is null ? null : new[]{Key};
    set => Key = Keys?.FirstOrDefault();
}
```